clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/security/EcsCredentialsInitializer.java

```diff
@@ -40,20 +41,24 @@ public ECSCredentialsConfig ecsCredentialsConfig() {
   }
 
   @Bean
-  @DependsOn("netflixAmazonCredentials")
+  @DependsOn("amazonCredentialsLoader")
   public List<? extends NetflixAmazonCredentials> netflixECSCredentials(
-      CredentialsLoader<? extends NetflixAmazonCredentials> credentialsLoader,
-      ECSCredentialsConfig credentialsConfig,
-      AccountCredentialsRepository accountCredentialsRepository)
+      CredentialsRepository<NetflixAmazonCredentials> accountCredentialsRepository,
```

Comment:
nitpick but can we call it `credentialsRepository` instead of `accountCredentialsRepository` - we're trying to be clear with the terms used.